src/web/services/userAccount.ts

```diff
@@ -12,7 +13,7 @@ import { backendError } from '../utils/apiError';
 
 export type UserAccount = {
   profile: KeycloakProfile;
-  user: UserWithIsUid2Support | null;
+  user: UserWithIsUid2Support | UserWithCurrentParticipantRoleNames | null;
```

Comment:
Is there a reason why we kept `UserWithisUid2Support` and `UserWithCurrentParticipantRoleNames` separate? Could we make one object that has the UserDTO + `isUid2Support` + `currentParticipantRoleNames` 

Then this could be the only type in UserAccount for LoggedInUser and it would make the code plus testing simpler I believe

Reply:
nah there's no reason - please fix it up :D

Ideally the type we use for `LoggedInUser` in the `CurrentUserContext` should have everything we need role-wise directly, so we won't need to do stuff like: `const user = LoggedInUser?.user as UserWithIsUid2Support;` in `manageParticipants.tsx`. 

I also realised that `UserWithCurrentParticipantRoleNames` also has the wrong name now, given it has the whole user role object, not just the name.